### ISSUE TITLE
Black Screen on Launch of Hyper [Windows 10]

### ISSUE THREAD

**Issue by crunchprank** [zeit/hyper#3758]
## Info

- **OS version and name**: Windows 10 64-bit Version 1809 OS Build 17763.615

- **Hyper.app version**: 3.0.2 (stable)

- **Hyper Config**: https://pastebin.com/raw/F747wqxU

- **Relevant information from devtools**: 

```
Uncaught TypeError: Cannot destructure property `uid` of 'undefined' or 'null'.
    at EventEmitter.u.a.on (bundle.js:1)
    at EventEmitter.emit (events.js:182)
    at t.a.ipcListener (bundle.js:1)
    at EventEmitter.emit (events.js:182)
```

- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue

I haven't done anything other than go to the official `hyper.is` website, download the Windows executable file, perform the installation, launch Hyper, go to Plugins -> Install Hyper CLI.

Seeing as I've never used this program before, I'm not sure what to expect.

Basically I launch Hyper which by default opens the Windows command prompt. I type `hyper` (which I'm assuming I'm supposed to type to interact with the `hyper cli`), it opens up a black window with a magenta cursor, and then nothing. I cannot type in the window. I provided the output above from the Developer Tools console. [Here is a screenshot](https://i.imgur.com/Hnau4Ga.png) of the Hyper window and dev console side by side.

Searching exisiting bugs that seem to be similar, all resolutions pointed to uninstalling or disabling plugins. However the only plugin I have touched is the official Hyper CLI one.

Any help would be appreciated.

**Comment by patroclos** [comment]
I have the same problem, also when starting hyper via the startmenu item the cmd works, but it doesn't load any plugins.

**OS**: Win 10 18362.239 Hyper 3.0.2

**Comment by binaryfunt** [comment]
I get the same problem, only when I try opening a repository in Hyper from the GitHub desktop app (but I don't get an uncaught type error in the developer console). Hyper opens normally when I open it from the Start menu.

Windows 10 64-bit v1903
Hyper 3.0.2 (stable) with no plugins

It doesn't matter if I set `shell` to `'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe'` or leave it empty in `.hyper.js`

**Comment by joonas-fi** [comment]
My Hyper screen went black after it worked for many days straight. Just this morning when I came back, it's all black:

![image](https://user-images.githubusercontent.com/630151/66988105-a1325080-f0ca-11e9-9ded-c6f0dfd5df2f.png)

Diagnostics:

- I'm on Win10, Hyper v3.02 (stable) with plugin: hyperpower v1.1.1
- F12 doesn't do anything (does not launch devtools)
- Clicking on where the close/minimize buttons should be, does not do anything
- Clicking on where the menu button should be, does not do anything
- Hyper doesn't use any considerable CPU (flickering between 0 % - 0.6 %)
- Restarting GPU does not help (Win+Ctrl+Shift+B)
- Killing one of the Hyper's own subprocesses killed the entire process group
- I started new instance of Hyper, and it works normally

**Comment by dpbeaumont** [comment]
I get the same error: Cannot destructure property `uid` of 'undefined' or 'null'. from the console in Developer tools.
Steps to replicate this for me, typing ` hyper .`  inside a powershell terminal, to open up the current directory, this was working a few months back.
